Translations/French/README.md

```diff
@@ -431,3 +460,23 @@ AddHandler application/x-httpd-php55 .php
 # Autrement, vous pouvez utiliser AddType
 AddType application/x-httpd-php55 .php
 ```
+### Désactiver la Compatibilité De Vue pour Internet Explorer
+La Compatibilité de vue dans IE peut affecter l'affichage de certains sites web. L'extrait suivant devrait forcer  IE d'utiliser le moteur d'interprétation Edge et desactiver disable la Compatibilité De Vue.
```

Comment:
The rule is "forcer quelqu'un à faire quelque chose, et, (être) forcé de faire quelque chose".
So here *forcer IE à utiliser*
and *d**é**sactiver**, delete disable